MSCL/source/mscl/MicroStrain/MIP/MipNode_Impl.cpp

```diff
@@ -2630,6 +2630,8 @@ namespace mscl
         {
             try
             {
+                const uint64_t originalTimeout = timeout();
```

Comment:
Shouldn't this be before the loop and then reset it after the try/catch?

Reply:
It could be done that way, but I would have to change a couple other things.

The way this works now means that the timeout is only increased for the two commands that need it, so if there is another issue where other commands are timing out, it won't take an extra 600 milliseconds per command.

Since the `timeout()` function is essentially just copying a number into another number I didn't think it was too bad of a performance hit to run this line everytime